client/checkout/woopay/connect/woopay-connect.js

```diff
@@ -2,7 +2,7 @@
  * Internal dependencies
  */
 import { WooPayConnectIframe } from 'wcpay/checkout/woopay/connect/woopay-connect-iframe';
-import ReactDOM from 'react-dom';
+import { createRoot } from 'react-dom/client';
```

Comment:
See https://react.dev/blog/2022/03/08/react-18-upgrade-guide#updates-to-client-rendering-apis